pkg/networkserver/errors.go

```diff
@@ -35,13 +35,11 @@ var (
 	errEncryptMAC                = errors.DefineInternal("encrypt_mac", "failed to encrypt MAC commands")
 	errFCntTooHigh               = errors.DefineInvalidArgument("f_cnt_too_high", "FCnt is too high")
 	errGatewayServerNotFound     = errors.DefineNotFound("gateway_server_not_found", "Gateway Server not found")
-	errInvalidADRMargin          = errors.DefineInvalidArgument("adr_margin", "invalid ADR margin")
 	errInvalidChannelIndex       = errors.DefineInvalidArgument("channel_index", "invalid channel index")
-	errInvalidClassBTimeout      = errors.DefineInvalidArgument("class_b_timeout", "invalid class B timeout")
-	errInvalidClassCTimeout      = errors.DefineInvalidArgument("class_c_timeout", "invalid class C timeout")
 	errInvalidConfiguration      = errors.DefineInvalidArgument("configuration", "invalid configuration")
 	errInvalidDataRate           = errors.DefineInvalidArgument("data_rate", "invalid data rate")
 	errInvalidFieldMask          = errors.DefineInvalidArgument("field_mask", "invalid field mask")
+	errInvalidFieldValue         = errors.DefineInvalidArgument("field_value", "invalid value of field `{field}`")
```

Comment:
These kind of changes should result in changes in `messages.json`, but I don't see anything in this PR. Also, Travis doesn't complain about it, so please check that.